src/main/resources/application.yml

```
@@ -10,7 +10,7 @@ spring:
     open-in-view: false
 
 
---- #dev
+--- # dev
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)